packages/mobile/src/tokens/selectors.ts

```diff
@@ -60,3 +61,23 @@ export const defaultTokenSelector = createSelector(tokensListSelector, (tokens)
 
   return maxTokenAddress
 })
+
+export const totalTokenBalanceSelector = createSelector(
```

Comment:
Can you add a test for this selector? I think there should be some examples in the selectors.test file :)

Reply:
added!